trio/_ssl.py

```diff
@@ -1,3 +1,18 @@
+from __future__ import annotations
```

Comment:
I don't think moving the import block above the comment is necessary. (future imports can be after comments IIRC)

Reply:
This happened after running isort. Correct, __future__ can be after only module docstrings and `#` comments.

Reply:
Didn't realize isort did that (I assumed it would keep everything under a comment as long as everything before it was under the comment), alright!